packages/now-static-build/src/index.ts

```diff
@@ -276,11 +311,19 @@ export async function build({
     console.log('Installing dependencies...');
     await runNpmInstall(entrypointDir, ['--prefer-offline'], spawnOpts, meta);
 
-    if (meta.isDev && pkg.scripts && pkg.scripts[devScript]) {
+    if (
+      meta.isDev &&
+      ((pkg && devScript && pkg.scripts && pkg.scripts[devScript]) ||
+        config.devCommand)
```

Comment:
Nit pick: Can you swap these two so that `config.devCommand` comes first before pkg script?

I think this will start to become more common if we recommend that.